### PR TITLE
addresses issue #110 add git reflog

### PR DESCRIPTION
**Pull request by finanalyst**

### CODE REVIEW

**Comment by finanalyst** [comment]
The effect of this can be seen on [new-raku](new-raku.finanalyst.org) go to footer, popup 'generated'
Please review the effect. Maybe reword the github related text??

**Comment by coke** [comment]
Currently shows:

```
This page is generated from

Website/structure-sources/index.rakudoc

on 2023-02-19 at 22:18:40

Raku source files upto commit ｢305537cc6｣, see [github.com/Raku/doc/tree/305537cc6](https://github.com/Raku/doc/tree/305537cc6)
```

I think this can be shorted to:

```
This page is generated from

Website/structure-sources/index.rakudoc

on 2023-02-19 at 22:18:40, from rev <a href="https://github.com/Raku/doc/tree/305537cc6">305537cc6</a>
```

**Comment by finanalyst** [comment]
I changed, see [new-raku](new-raku.finanalyst.org)
@coke please look again. 
I did this yesterday and forgot to update this thread. Also please look at the RakuTM next to it

**Comment by coke** [comment]
Yup, this looks good.
